### PR TITLE
Raw access to the query parameter string

### PR DESCRIPTION
**Pull request by thibaultmeyer**
When dealing with external software, sometime we have to get the raw parameter string (query string) to work with.

This PR allow developer to get this data easily.

### CODE REVIEW

**Review by asolntsev — APPROVED** [review]

File: ninja-servlet/src/main/java/ninja/servlet/NinjaServletContext.java [hunk 1]
```diff
@@ -140,6 +140,11 @@ public String getRequestUri() {
         return httpServletRequest.getRequestURI();
     }
 
+    @Override
+    public String getRequestParameterString() {
+        return httpServletRequest.getQueryString();
```
Comment (raphaelbauer) [inline ninja-servlet/src/main/java/ninja/servlet/NinjaServletContext.java:145]: I think we should write a test that makes sure we do not break the functionality in the future... I am sure there is a way to test this :)

File: ninja-core/src/main/java/ninja/Context.java [hunk 2]
```diff
@@ -107,6 +107,13 @@ private HTTP_STATUS(int code) {
     @Deprecated
     String getRequestUri();
 
+    /**
+     * Returns the parameter string that is contained in the request URL after the path.
+     *
+     * @return The parameter string
+     */
+    String getRequestParameterString();
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/Context.java:119]: Could you give an example in the Javadoc? Then it is even more straight-forward for everyone what this method does :)

**Comment by raphaelbauer** [comment]
Looks good :) Just some tiny remarks :)

File: ninja-core/src/main/java/ninja/Context.java (hunk 2 above)
Comment (thibaultmeyer) [reply ninja-core/src/main/java/ninja/Context.java:119]: Done

File: ninja-servlet/src/main/java/ninja/servlet/NinjaServletContext.java (hunk 1 above)
Comment (thibaultmeyer) [reply ninja-servlet/src/main/java/ninja/servlet/NinjaServletContext.java:145]: Done

File: ninja-core/src/main/java/ninja/Context.java [hunk 3]
```diff
@@ -107,6 +107,17 @@ private HTTP_STATUS(int code) {
     @Deprecated
     String getRequestUri();
 
+    /**
+     * Returns the parameter string that is contained in the request URL after the path.
+     * <br/>
+     * <b>Example:</b>
+     * For the URL <em>"http://domain.local/?key=private&hash=%93%af%f2%3b"</em>, this method
+     * will return <em>"key=private&hash=%93%af%f2%3b"</em>
```
Comment (raphaelbauer) [inline ninja-core/src/main/java/ninja/Context.java:115]: 👍

**Comment by raphaelbauer** [comment]
Nice!

**Pull request merged**